apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,243 @@
+                        borderRadius="4px 4px 0px 0px"
+                        boxShadow="0px 0px 20px 0px rgba(104, 104, 104, 0.30)"
+                        alt={`Image of ${e.layoutDisplayName} layout`}
+                      />
+
+                      {selectedLayout.layoutTypename !== e.layoutTypename && (
+                        <Box
+                          className="hover-text"
+                          position="absolute"
+                          top="0"
+                          left="0"
+                          right="0"
+                          bottom="0"
+                          bg="rgba(172, 199, 250, 0.4)"
+                          opacity="0"
+                          alignItems="center"
+                          justifyContent="center"
+                          borderRadius="6px" // due to this being smaller than the outer Box
+                        >
+                          <VStack
+                            w="100%"
+                            h="100%"
+                            justify="center"
+                            gap="0.25rem"
+                            opacity="1"
+                            color="base.content.brand"
+                          >
+                            <Icon as={BiShow} />
+                            <Text textStyle="caption-1">Click to preview</Text>
+                          </VStack>
+                        </Box>
+                      )}
+                    </Box>
+                    <HStack w="100%" justifyContent="space-between">
+                      <Text
+                        className="layout-title"
+                        textAlign="left"
+                        textStyle="h6"
+                        textColor={
+                          selectedLayout.layoutTypename === e.layoutTypename
+                            ? 'base.content.brand'
+                            : 'base.content.strong'
+                        }
+                        mb="0.5rem"
+                      >
+                        {e.layoutDisplayName} layout
+                      </Text>
+                    </HStack>
+                    <Text
+                      textAlign="left"
+                      textStyle="body-1"
+                      textColor="base.content.default"
+                    >
+                      {e.layoutDescription}
+                    </Text>
+                  </VStack>
+                )
+              })}
+            </VStack>
+          </GridItem>
+          <GridItem
+            w="100%"
+            colSpan={3}
+            bg="grey.100"
+            p="2rem 2rem 0 2rem"
+            overflow="auto"
+          >
+            <Box
+              borderRadius="8px 8px 0px 0px"
+              w="100%"
+              minH="100%"
+              boxShadow="0 0 20px 0 rgba(104,104,104,0.3)"
+              position="relative"
+            >
+              <HStack
+                borderRadius="8px 8px 0px 0px"
```

Comment:
nit: shorthand

Reply:
similar to above, seems like most compact form, since for border-radius 2 value shorthand sets diagonally opposite corners.